src/Model/Model.ts

```diff
@@ -710,6 +710,12 @@ export class Model<T extends IModelData> implements IModel<T> {
     return !this.isShadow && this.modelData.id && this.modelData.id.startsWith("new");
   }
 
+  /** Create a clone of the model without any of the unsaved changes */
+  public original() {
```

Comment:
Per my comment in the interface, consider making this a readonly property.
This means instead of `public original() { `, this line would be `public get original() {`

Reply:
I decided not to do that, I explained it in a comment on the front of the pr